pkg/config/config.go

```diff
@@ -17,9 +17,9 @@ const (
 	MutatingWebhookConfigurationDebugName = "kyverno-resource-mutating-webhook-cfg-debug"
 	MutatingWebhookName                   = "nirmata.kyverno.resource.mutating-webhook"
 
-	// ValidatingWebhookConfigurationName  = "kyverno-validating-webhook-cfg"
-	// ValidatingWebhookConfigurationDebug = "kyverno-validating-webhook-cfg-debug"
-	// ValidatingWebhookName               = "nirmata.kyverno.policy-validating-webhook"
+	ValidatingWebhookConfigurationName      = "kyverno-validating-webhook-cfg"
+	ValidatingWebhookConfigurationDebugName = "kyverno-validating-webhook-cfg-debug"
```

Comment:
To make the name consistent with mutating webhook, it would be nice to name the validatingwebhook to "kyverno-resource-validating-webhook-cfg" and "kyverno-resource-validating-webhook-cfg-debug".

Reply:
Requested changes have been made